shared/lib_irradproc.cpp

```diff
@@ -1084,11 +2259,14 @@ int irrad::calc()
 	{
 		double sunanglestemp[9];
 		if (day > 1) //simply decrement day during month
-			solarpos(year, month, day - 1, 12, 0.0, latitudeDegrees, longitudeDegrees, timezone, sunanglestemp);
+			//solarpos(year, month, day - 1, 12, 0.0, latitudeDegrees, longitudeDegrees, timezone, sunanglestemp);
+			solarpos_spa(year, month, day - 1, 12, 0.0, 0.0, latitudeDegrees, longitudeDegrees, timezone, 0, 66.7, elevation, pressure, temp, tiltDegrees, surfaceAzimuthDegrees, sunanglestemp);
```

Comment:
If `dut1` is always 0 and `delta_t` is always 66.7, please set these as constants and explain in a comment why, similar to this: https://github.com/NREL/ssc/blob/728c55c9cef1781af1ee946d76542efead86f0fe/shared/lib_irradproc.h#L342

Reply:
Updated delta_t to be calculated value based on year, set dut1 as static const value of 0 (leap second, assumed not to be included in analysis).